gluoncv/model_zoo/mask_rcnn/mask_rcnn.py

```diff
@@ -540,3 +550,119 @@ def mask_rcnn_fpn_resnet101_v1d_coco(pretrained=False, pretrained_base=True, **k
         rpn_test_pre_nms=6000, rpn_test_post_nms=1000, rpn_min_size=0,
         num_sample=512, pos_iou_thresh=0.5, pos_ratio=0.25, deep_fcn=True,
         **kwargs)
+
+
+def mask_rcnn_fpn_resnet18_v1b_coco(pretrained=False, pretrained_base=True, **kwargs):
+    r"""Mask RCNN model from the paper
+    "He, K., Gkioxari, G., Doll&ar, P., & Girshick, R. (2017). Mask R-CNN"
+
+    Parameters
+    ----------
+    pretrained : bool or str
+        Boolean value controls whether to load the default pretrained weights for model.
+        String value represents the hashtag for a certain version of pretrained weights.
+    pretrained_base : bool or str, optional, default is True
+        Load pretrained base network, the extra layers are randomized. Note that
+        if pretrained is `True`, this has no effect.
+    ctx : Context, default CPU
+        The context in which to load the pretrained weights.
+    root : str, default '~/.mxnet/models'
+        Location for keeping the model parameters.
+
+    Examples
+    --------
+    >>> model = mask_rcnn_fpn_resnet18_v1b_coco(pretrained=True)
+    >>> print(model)
+    """
+    from ..resnetv1b import resnet18_v1b
+    from ...data import COCODetection
+    classes = COCODetection.CLASSES
+    pretrained_base = False if pretrained else pretrained_base
+    base_network = resnet18_v1b(pretrained=pretrained_base, dilated=False, use_global_stats=True)
+    features = FPNFeatureExpander(
+        network=base_network,
+        outputs=['layers1_relu3_fwd', 'layers2_relu3_fwd', 'layers3_relu3_fwd',
+                 'layers4_relu3_fwd'], num_filters=[256, 256, 256, 256], use_1x1=True,
+        use_upsample=True, use_elewadd=True, use_p6=True, no_bias=False, pretrained=pretrained_base)
+    top_features = None
+    box_features = nn.HybridSequential()
+    box_features.add(nn.AvgPool2D(pool_size=(3, 3), strides=2, padding=1))  # reduce to 7x7
+    for _ in range(2):
+        box_features.add(nn.Dense(1024, weight_initializer=mx.init.Normal(0.01)),
+                         nn.Activation('relu'))
+    train_patterns = '|'.join(['.*dense', '.*rpn', '.*mask', 'P',
+                               '.*down(2|3|4)_conv', '.*layers(2|3|4)_conv'])
+    return get_mask_rcnn(
+        name='fpn_resnet18_v1b', dataset='coco', pretrained=pretrained,
+        features=features, top_features=top_features, classes=classes,
+        box_features=box_features, mask_channels=256, rcnn_max_dets=1000,
+        short=800, max_size=1333, min_stage=2, max_stage=6,
+        train_patterns=train_patterns, nms_thresh=0.5, nms_topk=-1,
+        post_nms=-1, roi_mode='align', roi_size=(14, 14),
+        strides=(4, 8, 16, 32, 64), clip=4.42, rpn_channel=1024, base_size=16,
+        scales=(2, 4, 8, 16, 32), ratios=(0.5, 1, 2), alloc_size=(384, 384),
+        rpn_nms_thresh=0.7, rpn_train_pre_nms=12000, rpn_train_post_nms=2000,
+        rpn_test_pre_nms=6000, rpn_test_post_nms=1000, rpn_min_size=0,
```

Comment:
We need to discuss for a way to expose the `rpn_test_pre_nms` and `rpn_test_post_nms` params. Otherwise the small models won't be so fast anyway.